backend/src/routes.js

```diff
@@ -3,15 +3,28 @@ const router = express.Router()
 const { auth, guard, createToken } = require('./auth')
 
 const account = require('./controllers/login')(createToken)
+const notification = require('../controllers/notificationController');
 
-router.use(auth.unless({path: ['/login']}))
+router.use(auth.unless({ path: ['/login'] }))
 
 router.post('/login', account.login)
 router.get('/account', account.account)
 
+//Notifications system
+router.route('/user/:user/notifications')
+    .get(notification.getNotifications)
+    .post(notification.createNotification);

```

Comment:
I don't think we should let the client create notifications with an endpoint. I think we should just create them internally when someone delivers waste

Reply:
Make sense